.eslintrc.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
     ecmaVersion: 2020,
   },
   plugins: ['jest', 'import', 'eslint-plugin-header'],
+  ignorePatterns: ['LICENSE'],
```

Comment:
Why is this necessary? As I understand some script was run to update all the licenses for all of the files?